apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -5,24 +5,34 @@ import { Navbar } from '../../../components/Navbar.html.js';
 
 const examplePrompts = [
   {
-    id: 'random_mc',
-    question:
-      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all of the numbers to the user, and then make each of the numbers a potential answer, with the median being the correct answer.',
+    id: 'Select median of 5 random numbers',
+    promptGeneral:
+      'Write a multiple choice question asking the user to choose the median of 5 random numbers between 1 and 100. Display all numbers to the user, and ask them to choose the median.',
+    promptUserInput:
+      'Each random number generated should be a potential answer to the multiple-choice question. Randomize the order of the numbers.',
+    promptGrading: 'Make the actual median the correct answer.',
   },
   {
-    id: 'random_int',
-    question:
-      'Write a question that asks the user to multiply two integers. You should randomly generate two integers A and B, display them to the user, and then ask the user to provide the product C = A * B in an integer input box. The correct answer should be the product of the two numbers that you generated.',
+    id: 'Sum random integers',
+    promptGeneral:
+      'Write a question that asks the user to multiply two integers. You should randomly generate two integers A and B, display them to the user, and then ask the user to provide the product C = A * B.',
+    promptUserInput: 'Provide an integer input box for the user to enter the product.',
+    promptGrading: 'The correct answer is the actual product',
   },
   {
-    id: 'basic_int',
-    question:
-      'Write a question asking "What The Answer to the Ultimate Question of Life, the Universe, and Everything?". Provide an integer box for the user to answer. The correct answer is 42.',
+    id: 'Answer to Ultimate Question',
+    promptGeneral:
+      'Write a question asking "What Is The Answer to the Ultimate Question of Life, the Universe, and Everything?".',
+    promptUserInput: 'Provide an integer box for the user to answer.',
+    promptGrading: 'The correct answer is 42.',
   },
   {
-    id: 'implicit_specified_physics',
-    question:
+    id: 'Calculate Projectile Distance',
+    promptGeneral:
       'Write a question that asks the user to calculate how far a projectile will be launched. Display to the user an angle randomly generated between 30 and 60 degrees, and a velocity randomly generated between 10 and 20 m/s, and ask for the distance (in meters) that the object travels assuming no wind resistance.',
+    promptUserInput: 'Provide a numerical input box for the user to enter an answer.',
+    promptGrading:
+      'The correct answer is the distance that the projectile will travel, using the corresponding formula',
```

Comment:
Nit for consistency:

```suggestion
      'The correct answer is the distance that the projectile will travel, using the corresponding formula.',
```